### ISSUE TITLE
Missing py.typed Marker

### ISSUE THREAD

**Issue by lvieirajr** [danielfm/pybreaker#84]
Hello, thanks for the great package.
When using it with mypy I will get the missing py.typed marker as this is not being exported alongside the package;
https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports

```
Skipping analyzing "pybreaker": module is installed, but missing library stubs or py.typed marker [import] See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

Any way that could be included with the next version?

**Comment by danielfm** [comment]
Hi. 

I'm not familiar with mypy, so do you have the concrete actions on how to fix the issue, or better yet, are you willing to submit a fix for this?

**Comment by martijnthe** [comment]
See https://github.com/danielfm/pybreaker/pull/86

**Comment by danielfm** [comment]
I just published 1.0.3 that fixes this issue.

Thanks @martijnthe!
